### DISCUSSION TITLE
Is there a way to make Nuxt 3 SSR only?

### DISCUSSION THREAD

**Discussion by xlmnxp** [nuxt/nuxt#18759] [Questions]
Hello,
I have nuxt project and I want to use NodeJS modules in it but I don't want it to render in front-end/client side
only serve HTML + CSS to client and render everything in Back-End (something like ASTRO node adapter)

**Comment by danielroe** [comment]
Yes, there is! You can set `experimental.noScripts` in your Nuxt config file.

**Reply by xlmnxp** [reply]
Will components will still interactive from SSR?
for example if I click increment button in FrontEnd, will it call BackEnd and get response back?

**Reply by xlmnxp** [reply]
@danielroe something like Vaadin if you know it
it render front-end and update it from back-end
https://vaadin.com/

**Reply by TheAlexLichter** [reply]
@xlmnxp You might look for server components https://github.com/nuxt/nuxt/commit/2d013c5 👀

**Reply by xlmnxp** [reply]
@manniL I didn't understand that feature well :(
Is it require Client components to pass props to it?
